mlflow/utils/mlflow_tags.py

```diff
@@ -1,18 +1,27 @@
 """
 File containing all of the run tags in the mlflow. namespace.
 """
+
 MLFLOW_DATABRICKS_NOTEBOOK_ID = "mlflow.databricks.notebookID"
 MLFLOW_DATABRICKS_NOTEBOOK_PATH = "mlflow.databricks.notebookPath"
 MLFLOW_DATABRICKS_WEBAPP_URL = "mlflow.databricks.webappURL"
 MLFLOW_DATABRICKS_RUN_URL = "mlflow.databricks.runURL"
 MLFLOW_DATABRICKS_SHELL_JOB_ID = "mlflow.databricks.shellJobID"
 MLFLOW_DATABRICKS_SHELL_JOB_RUN_ID = "mlflow.databricks.shellJobRunID"
+
 MLFLOW_RUN_NAME = "mlflow.runName"
-MLFLOW_GIT_BRANCH_NAME = "mlflow.gitBranchName"
-MLFLOW_GIT_REPO_URL = "mlflow.gitRepoURL"
+MLFLOW_SOURCE_TYPE = "mlflow.source.type"
+MLFLOW_SOURCE_NAME = "mlflow.source.name"
+MLFLOW_GIT_COMMIT = "mlflow.source.git.commit"
+MLFLOW_GIT_BRANCH = "mlflow.source.git.branch"
+MLFLOW_GIT_REPO_URL = "mlflow.source.git.repoURL"
 MLFLOW_PARENT_RUN_ID = "mlflow.parentRunId"
 MLFLOW_ENV = "mlflow.project.env"
+MLFLOW_PROJECT_ENTRY_POINT = "mlflow.project.entryPoint"
 MLFLOW_DOCKER = "docker"
 MLFLOW_CONDA = "conda"
 MLFLOW_DOCKER_IMAGE_NAME = "mlflow.docker.image.name"
 MLFLOW_DOCKER_IMAGE_ID = "mlflow.docker.image.id"
+
+LEGACY_MLFLOW_GIT_BRANCH_NAME = "mlflow.gitBranchName"
```

Comment:
Could we add a comment on these (e.g., these were the tag names prior to 0.9.0, and we want to keep backwards-compatibility for at least one release)?

Reply:
Done.